Cargo.toml

```diff
@@ -11,6 +11,7 @@ documentation = "http://doc.servo.org/smallvec/"
 
 [features]
 heapsizeof = ["heapsize", "std"]
+serde_support = ["serde"]
```

Comment:
The Rust API guidelines [recommend][1] using just the crate name `serde`, rather than adding a separate feature name.

[1]: https://rust-lang-nursery.github.io/api-guidelines/interoperability.html#crate-has-a-serde-cfg-option-that-enables-serde-c-serde-cfg